docs/ReleaseNotes.md

```diff
@@ -1,7 +1,12 @@
 # Release Notes
 
-## Unreleased
+## 2.5.x-prerelease
 
+- Adds a backlog/retry mechanism for commands issued while a connection isn't available (#1912 via NickCraver)
+  - Commands will be queued if a multiplexer isn't yet connected.
+  - Commands will be queued if a connection is lost, for retry when it reconnects.
```

Comment:
semantic nit: it's not exactly "retry"... more like "Commands will be queued if a connection is lost, and then sent to the server when the connection is restored"